lib/Profile.dart

```diff
@@ -90,9 +97,11 @@ class _ProfileState extends State<Profile> {
                                   " " +
                                   userInfoProvider.info.lastName,
                               style: TextStyle(
+                                fontFamily: 'SFDisplay',
                                 fontSize: 22,
```

Comment:
Is this not the same as title1? Or is this unique enough to have a TextStyle here

Reply:
Discussed offline: Profile isn't integrated with the design system on Figma, will update when it is